chsdi/templates/htmlpopup/aktuelle_erdbeben.mako

```diff
@@ -0,0 +1,44 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<%
+    import requests
+    request = context.get('request')
+    protocol = request.scheme
+    defaultCoord = ['600000', '200000']
+    defaultBbox = ['600000', '200000', '650000', '250000']
+    topic = request.matchdict.get('map')
+    baseUrl = request.registry.settings['api_url']
+
+    # generate getfeatureinfo request
+    coord_x, coord_y = [float(x) for x in request.params['coord'].split(",")] if request.params['coord'] else defaultCoord
+    bbox_xmin, bbox_ymin, bbox_xmax, bbox_ymax = [float(x) for x in request.params['mapExtent'].split(",")] if request.params['mapExtent'] else defaultBbox
+    imageDisplay = [int(x) for x in request.params['imageDisplay'].split(",")]
+    getfeatureinfo_x = int(((coord_x - bbox_xmin)/(bbox_xmax-bbox_xmin))*imageDisplay[0])
+    getfeatureinfo_y = int(((bbox_ymax - coord_y)/(bbox_ymax-bbox_ymin))*imageDisplay[1])
```

Comment:
remember html popup is supposed to work without any extra parameter. instead try using the feature geometry if available instead as this will fail otherwise.